pkg/jaeger/query/query.go

```diff
@@ -41,45 +44,83 @@ func (p *Query) SpanWriter() spanstore.Writer {
 }
 
 func (p *Query) GetTrace(ctx context.Context, traceID model.TraceID) (*model.Trace, error) {
+	metrics.RequestsTotal.With(prometheus.Labels{"type": "trace", "handler": "get_trace", "code": ""}).Inc()
+	start := time.Now()
 	res, err := getTrace(ctx, p.conn, traceID)
 	if err == nil {
+		metrics.RequestsTotal.With(prometheus.Labels{"type": "trace", "handler": "get_trace", "code": "200"}).Inc()
 		traceRequestsExec.Add(1)
+		metrics.RequestsDuration.With(prometheus.Labels{"type": "trace", "handler": "get_trace"}).Observe(time.Since(start).Seconds())
```

Comment:
Should we add code 200 here as well? Just so its clear that this is successful requests only.